pkg/schemaconv/testdata/preserve-unknown.yaml

```diff
@@ -2,31 +2,15 @@ types:
 - name: io.k8s.testcase.Empty
   map:
     elementType:
-      scalar: untyped
-      list:
-        elementType:
-          namedType: __untyped_atomic_
-        elementRelationship: atomic
-      map:
-        elementType:
-          namedType: __untyped_deduced_
-        elementRelationship: separable
+      namedType: __untyped_deduced_
 - name: io.k8s.testcase.EmptyPreserveUnknownFieldsObject
   map:
     fields:
     - name: preserveField
       type:
         map:
```

Comment:
Did we have `map` twice in that? That looks super bogus?

Reply:
You are talking about `io.k8s.testcase.EmptyPreserveUnknownFieldsObject`? Looks right to me? Source OpenAPI:

```json
    "io.k8s.testcase.EmptyPreserveUnknownFieldsObject": {
      "description": "",
      "properties": {
        "preserveField": {
          "description": "",
          "type": "object",
          "x-kubernetes-preserve-unknown-fields": true
        }
      }
```

Reply:
I don't know if I'm reading the diff well, but it looks like the `io.k8s.testcase.EmptyPreserveUnknownFieldsObject` had two "maps" field, which seems very wrong?

Reply:
I dont see it

Reply:
Sorry I was confused by the indentation, forget it, it's good (it always was, I was just surprised).